TestForSignUp/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Threading;
+
+namespace TestForSignUp
+{
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
Why test program in this PR??

Reply:
?

Reply:
Mb I don`t understand something. Is this stuff need for tests?

Reply:
do yo mean class program?

Reply:
yes

Reply:
yes, we need this